### PR TITLE
[Edit] Git: Pull Requests - Add Note 

### PR DESCRIPTION
**Pull request by liye0**
### Description

- Added note under Method 1: Fork explaining exactly when a fork is needed.

### Issue Solved

- Closes Issue #4377 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/git/concepts/pull-requests/pull-requests.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ PRs are featured on popular version control platforms such as [GitHub](https://g
 4. `add`, `commit`, and `push` the changes to your forked repository.
 5. From the web page of your forked repository, click on the "Pull requests" tab.
 
+> **Note**: Forking a repository is needed when the user lacks write access to the original repository or when the user wants to make changes without affecting the original repository.
```
Comment (ishg-153) [inline content/git/concepts/pull-requests/pull-requests.md:37]:
```suggestion
> **Note**: Forking a repository is needed when the user lacks write access to the original repository or wants to make changes without affecting the original repository.
```

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@liye0 I've suggested one small change.

**Review by helfese — APPROVED** [review]

**Comment by liye0** [comment]
@ishg-153 Could you please review the change

**Review by ishg-153 — APPROVED** [review]
@liye0 Looks good for a second review!

**Review by avdhoottt — APPROVED** [review]
LGTM!!

**Comment by ishg-153** [comment]
@avdhoottt merging this!

**Pull request merged**